### PR TITLE
Reword for clarity

### PR DESCRIPTION
**Pull request by earthrid**
Reworded one sentence for clarity and ease of reading.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The Guides are primarily meant to cover the "Ember happy path", and are not inte
...
 
-Liberally use examples in your writing. For example, the sentence "Templates can contain expressions in double curly braces." should have added to it "Templates can contain expressions in double curly braces, like \`&lt;h1&gt;Welcome {{user.name}}&lt;/h1&gt;\`." A short example is often more clear than a long explanation. In fact, it is often helpful to give two examples to make things even clearer.
+Liberally use examples in your writing. For example, the sentence "Templates can contain expressions in double curly braces" could be expanded to "Templates can contain expressions in double curly braces, such as \`&lt;h1&gt;Welcome {{user.name}}&lt;/h1&gt;\`" A short example is often more clear than a long explanation. In fact, it is often helpful to give two examples to make things even clearer.
```
Comment (locks) [inline CONTRIBUTING.md]: `should be expanded`

Comment (earthrid) [inline CONTRIBUTING.md]: @locks What the sentence should be expanded to will vary according to individual cases, but I understand the wish to have a "should" in the sentence. How about:
"should be expanded to something like"

Comment (locks) [inline CONTRIBUTING.md]: Sounds good to me, thanks :)

**Comment by locks** [comment]
Can you squash the commits please?

**Comment by earthrid** [comment]
Hi @locks, now done.

**Pull request merged**